docs/reference/NuGet-Client-SDK.md

```diff
@@ -55,6 +57,27 @@ Search for "json" packages using the [NuGet V3 Search API](../api/search-query-s
 
 [!code-csharp[SearchPackages](~/../nuget-samples/NuGetProtocolSamples/Program.cs?name=SearchPackages)]
 
+### Create a package
+
+Create a package, set metadata, and add dependencies using [`NuGet.Packaging`](https://www.nuget.org/packages/NuGet.Packaging).
+
+> [!IMPORTANT]
+> It is strongly recommended that NuGet packages are created using the official NuGet tooling and **not** using this
+> low-level API. There are a variety of characteristics important for a well-formed package and the latest version of
+> tooling helps incorporate these best practices.
+> 
+> For more information about creating NuGet packages, see the overview of the
+> [package creation workflow](../create-packages/overview-and-workflow.md) and the documentation for official pack
+> tooling (For example, [Using the dotnet CLI](../create-packages/creating-a-package-dotnet-cli.md)).
```

Comment:
The upper casing seems odd to me. What do you think of this?

```suggestion
> tooling (for example, [using the dotnet CLI](../create-packages/creating-a-package-dotnet-cli.md)).
```

I feel less strongly about `Using the dotnet CLI`, feel free to keep that as-is.

Reply:
@loic-sharma You're right about the "for example". The reference to another topic ("Using the...") is capitalized, just like the topic.